assets/js/src/core/modules/asset/editor/image/tab-manager/tabs/properties/components/table/cells/boolean-cell/boolean-cell.tsx

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Maybe we can move this file to the grid folder. For now this looks quite reusable. :)

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/324/commits/458d218286356d64030e6c25af5c0d3fe91beeaf